tests/output/InteractionsTest.py

```diff
@@ -12,11 +13,35 @@ class InteractionsTest(unittest.TestCase):
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
         section = Section('')
```

Comment:
I would just remove this variable and wouldn't call `fail_acquire_settings` with `section`, so just
```
self.assertRaises(TypeError, fail_acquire_settings, log_printer, None)
```
Same below.

PS: Ignore the "unnecessary change" comments then, as you have to modify these lines then :+1:

Reply:
Ah and `fail_acquire_settings(..., ..., None)` just becomes `fail_acquire_settings(..., ...)` (see occurrence below)